CONTRIBUTING.md

```diff
@@ -86,4 +86,4 @@ Example:
 
 1. Create a [new release](https://github.com/akka/alpakka/releases/new) with the next tag version (e.g. `v0.3`), title and release decsription including notable changes mentioning external contributors.
 2. Travis CI will start a [CI build](https://travis-ci.org/akka/alpakka/builds) for the new tag and publish artifacts to Bintray and will sync them to Maven Central.
-3. Checkout the newly created tag and run `sbt "deployRsync repo.akka.io"` to deploy API and reference documentation.
+3. Update the configuration of [Lightbend Tech Hub](https://github.com/typesafehub/tech-hub/blob/master/docsites.sbt#L70) to include the latest version.
```

Comment:
Does this mean that we have to manually create a PR to techhub for each new release?